### ISSUE TITLE
MandrillRequestDispatcher.execute throwing NPE

### ISSUE THREAD

**Issue by katangagonzalez** [rschreijer/lutung#88]
After upgrading to version 0.0.8 I started to have some `NPE` at line `124` of `MandrillRequestDispatcher`, this is because `response` is `null` at this point. Is there any workaround for this or a way to debug my request in order to figure out what is going on? Many thanks.

**Comment by katangagonzalez** [comment]
Please, check this [commit](https://github.com/rschreijer/lutung/commit/6fb898519568a183c4b5bbe3bd5ce76ab1e3f23f#diff-cddcf31baff09c25c4297296c7668c59) where `null` check was removed

**Comment by billoneil** [comment]
Hmm that commit looks like it could be the culprit but I'm not sure how the response is actually null there. Are you using a proxy or anything?

**Comment by katangagonzalez** [comment]
No, I assume that call to `execute` threw an exception. Would you recommend to come back to 0.0.7 version?

**Comment by ndemengel** [comment]
Hi, I'm experiencing the same problem for some requests. Could you please restore the null check?

**Comment by pv-pavan** [comment]
https://github.com/rschreijer/lutung/pull/103 This shud fix.
